scripts/git_functions.sh

```diff
@@ -569,13 +578,12 @@ function install_ros_noetic(){
     sudo apt update
     ic "> Installing ROS ... (might take a while)"
     apt_install -y ros-noetic-desktop-full
-    source /opt/ros/noetic/setup.zsh
+    ic_source /opt/ros/$ROS_DISTRO/setup.zsh "ROS_DISTRO=$ROS_DISTRO"
     ic_wrn "x- Done ROS installation."
 
     ic "> install Catkin Build tools ... "
     apt_install python3-pip
     sudo pip3 install -U catkin_tools
-    catkin config --extend /opt/ros/noetic
```

Comment:
@ajoardar007  I think this line caused the config tool to target the build at the home directory